### ISSUE TITLE
how to adapt to use in opera

### ISSUE THREAD

**Issue by vicanfon** [microsoft/vscode-chrome-debug-core#376]
Is there anyway to use debugger for chrome with opera browser?

**Comment by roblourens** [comment]
Do you know whether it supports chrome's remote debug protocol?

**Comment by DingBEN** [comment]
roblourens sounds like a bot... non-sequitur response
I have the same question, and after an extensive search on the Internet, ZERO, ZAPPO, not one answer that gives any sort of clue.
- VS Code debug for node.js refuses to connect
now thinking to stop wasting my time and find a like tool that people use, and works

**Comment by roblourens** [comment]
Not a non-sequitur. If it supports the chrome remote debug protocol, this library would support it. If not, you need something else.

**Comment by DingBEN** [comment]
t.y. for your response
- non-sequitur may not have been the exactly correct wording, but your answers leave a noob hanging
- I came to this page via a link that promised successful debugging(which is failing in my case)
  Have no idea what I need to make this work, nor at first understood this is a library or even why I would need a library on a tool(VS Code) that includes an integrated debugging feature.
- though by now I had encountered by luck what your response actually engenders.
- I still do not know what to do... except to continue to search, as your last answer is also from the perspective of one who is in the know
- yet you gave me a new clue when it concerns your library as I visited the <code> page and noticed the list of debuggers...

Thank you anyway.
